google-cloud-firestore/src/test/java/com/google/cloud/firestore/WatchTest.java

```diff
@@ -84,6 +86,10 @@
 
 @RunWith(MockitoJUnitRunner.class)
 public class WatchTest {
+
+  @Rule
+  public Timeout timeout = new Timeout(1, TimeUnit.SECONDS);
```

Comment:
While working through problem, I introduced bug that made unit tests hang. To make sure tests completed, I added timeout to fail test if they don't run within 1 second.